README.md

```diff
@@ -111,20 +104,6 @@ yarn kusama:start
 yarn polkadot:start
 ```
 
-### Test
-
-✅❌ Running tests locally:
-
-```bash
-yarn polkadot:test
-
-or
-
-yarn kusama:test
-```
-
-There is no need to run both as the tests are unified.
-
```

Comment:
Right now this runs [5 tests](https://github.com/w3f/polkadot-wiki/blob/master/tests/rpc.test.js) relating to react RPC components.  Given the limited coverage I don't think this needs to be on the main README.